projects/js-packages/publicize-components/src/components/social-previews/constants.js

```diff
@@ -25,12 +26,12 @@ export const AVAILABLE_SERVICES = [
 		name: 'facebook',
 		preview: FacebookPreview,
 	},
-	/* {
+	{
```

Comment:
As we discussed, can we wrap this in the Instagram feature check? It might help make sure that there's no confusion

Reply:
Done. Thanks